src/Annotation/AnnotationCollection.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-code for the canonical source repository
```

Comment:
license headers are not allowed anymore. 
using codestandard 2.2+ will verify that. 

would be good if you take cover on that manually in this PR.